engine/tester/fixtures/users_fixtures.go

```diff
@@ -68,7 +68,7 @@ var (
 		Paymails: []Paymail{
 			"recipient@" + PaymailDomainExternal,
 		},
-		PrivKey: "",
+		PrivKey: "xprvA8mj2ZL1w6Nqpi6D2amJLo4Gxy24tW9uv82nQKmamT2rkg5DgjzJZRFnW33e7QJwn65uUWSuN6YQyWrujNjZdVShPRnpNUSRVTru4cxaqfd",
```

Comment:
I believe that I made some assumption about this field in some initialisation that the user is external if it's empty but maybe you already came at this or maybe it was already changed.